functions/apiProxy/participant_data_handler.py

```
@@ -0,0 +1,875 @@
+
+        return {
+            'organization_id': claims.get('organization_id'),
+            'organization_name': claims.get('organization_name'),
+            'access_level': claims.get('access_level', 'basic'),
+            'can_access_pii': claims.get('pii_access', False),
+            'can_access_grades': claims.get('grade_access', False),
+            'course_access': claims.get('course_access', []),
+            'security_level': security_level
+        }
+    except Exception:
+        raise ValueError("Invalid JWT token")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)